.github/workflows/ci.yml

```
@@ -90,3 +90,16 @@ jobs:
           sudo apt install -y mesa-vulkan-drivers vulkan-validationlayers
 
       - run: cargo llvm-cov nextest --workspace --features mint,glam --fail-under-lines 80
+
+  docs:
```

```suggestion
  tests_miri:
```

I think this is the cause of the CI being stuck (there is already a `docs` job).